### ISSUE TITLE
i need help

### ISSUE THREAD

**Issue by dudethedude** [niftools/nifskope#113]
![capture](https://user-images.githubusercontent.com/29995163/27990711-16269b62-6414-11e7-9efe-dad4a5929b80.PNG)

what does any of this mean i thought i just export the obj and there you go, but i got this and tried searching on google about it and still no explanations on what they are and how to fix this, also tried redoing it but still i get the same message. if someone understands what to do please help

**Comment by hexabits** [comment]
@dudethedude 

This GitHub repo is not a support site.  Please stop creating issues because you don't know what you are doing.  That is what the forums are for. 

Anyway,  please actually read and parse the message.  It's in plain English.  "Entire scene will be exported" is pretty clear...  Just because something pops up does not mean it's an error that you need to "fix".
